vpr/CMakeLists.txt

```diff
@@ -25,11 +25,18 @@ file(GLOB_RECURSE LIB_SOURCES src/*/*.cpp)
 file(GLOB_RECURSE LIB_HEADERS src/*/*.h)
 files_to_dirs(LIB_HEADERS LIB_INCLUDE_DIRS)
 
+capnp_generate_cpp(CAPNP_SRCS CAPNP_HDRS
+    src/util/place_delay_model.capnp
+    src/util/matrix.capnp)
+
 #Create the library
 add_library(libvpr STATIC
+             ${CAPNP_SRCS}
              ${LIB_HEADERS}
-             ${LIB_SOURCES})
+             ${LIB_SOURCES}
+             )
 target_include_directories(libvpr PUBLIC ${LIB_INCLUDE_DIRS})
+target_include_directories(libvpr PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/src/util)
```

Comment:
Probably want to add a comment and clarify that any generated headers shouldn't be externally visible (I assume that's what this is for).